src/Microsoft.ML.Core/Data/UInt128.cs

```diff
@@ -11,14 +11,14 @@ namespace Microsoft.ML.Runtime.Data
     /// <summary>
     /// A sixteen-byte unsigned integer.
```

Comment:
Can you update this xml summary comment? This isn't really an "integer" anymore.